index.js

```diff
@@ -3,6 +3,15 @@
 const { setGlobalDispatcher, getGlobalDispatcher } = require('undici')
 const { fetch } = require('./src/fetch')
 
+fetch.Request = require('./src/request').Request
+fetch.Response = require('./src/response').Response
+fetch.Headers = require('./src/headers').Headers
+fetch.Body = require('./src/body').Body
+
+fetch.internals = {
+  symbols: require('./src/symbols')
```

Comment:
Not sure it’s a good idea to expose this

Reply:
i agree - removed